# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/calligra/hotfix-poppler.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- calligra-25.12.1/filters/karbon/pdf/SvgOutputDev.cpp.vanilla	2026-01-16 19:38:59.501500832 +0100
+++ calligra-25.12.1/filters/karbon/pdf/SvgOutputDev.cpp	2026-01-16 19:43:46.600896766 +0100
@@ -492,7 +492,11 @@
     Q_UNUSED(interpolate)
     Q_UNUSED(inlineImg)
     ImageStream *imgStr = new ImageStream(str, width, colorMap->getNumPixelComps(), colorMap->getBits());
+#if POPPLER_VERSION_MACRO < QT_VERSION_CHECK(26, 1, 0)
     imgStr->reset();
+#else
+    imgStr->rewind();
+#endif
 
     unsigned int *dest = nullptr;
     unsigned char *buffer = new unsigned char[width * height * 4];
